frontend/src/core/loaders/components/SkeletonLoader.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import './SkeletonLoader.css';
+
+const list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+const SkeletonLoader = () => (
+    <div className="skeleton-loader entities unselectable">
+        <ul>
+            { list.map((i) => {
+                const classes = `entity missing ${i ===1 ? 'selected' : null}`
```

Comment:
Please add space after `===` and change 1 to 0 now that the `list` will start from 0.